drivers/timer/Kconfig

```diff
@@ -39,6 +39,14 @@ config SYSTEM_CLOCK_INIT_PRIORITY
 	  the clock to be running already, you should let the default value as it
 	  is (0).
 
+config SYSTEM_CLOCK_MP_BROADCAST
+	bool "select whether or not the timer is broadcast"
+	default y if SMP
```

Comment:
Actually I'd go with n here and select by driver: right now I believe only the x86 timers (hpet, apic_timer and apic_tsc) and intel_adsp are truly broadcast.  The others implement the "deliver interrupt on CPU where sys_clock_set_timeout() was called" idiom, though we should get the owners to confirm.

Reply:
> Actually I'd go with n here and select by driver

done, select is tricky, found another way.